<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>653</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>SHARP Scriptr</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Canvas</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="toolBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>651</width>
      <height>111</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDeploy">
    <property name="title">
     <string>Deploy</string>
    </property>
    <addaction name="actionLocal_Deploy"/>
    <addaction name="actionUSB_Deploy"/>
    <addaction name="actionFTP_Deply"/>
    <addaction name="separator"/>
    <addaction name="actionDeploy_Options"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDeploy"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLocal_Deploy">
   <property name="text">
    <string>Local Deploy</string>
   </property>
  </action>
  <action name="actionUSB_Deploy">
   <property name="text">
    <string>USB Deploy</string>
   </property>
  </action>
  <action name="actionFTP_Deply">
   <property name="text">
    <string>FTP Deploy</string>
   </property>
  </action>
  <action name="actionDeploy_Options">
   <property name="text">
    <string>Deploy Settings...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
